hooks/contracts.ts

```diff
@@ -72,3 +73,10 @@ export function useSolSubdomainContract() {
     address: process.env.NEXT_PUBLIC_SOL_SUBDOMAINS,
   });
 }
+
+export function useMulticallContract() {
+  return useContract({
+    abi: multicall_abi.abi as Abi,
+    address: process.env.NEXT_PUBLIC_MULTICALL_CONTRACT,
```

Comment:
is this also on mainnet ?

Reply:
yes this contract is already deployed on mainnet, we use it in the auto renew bot